tests/assert_pass/mstore8.yul

```diff
@@ -0,0 +1,7 @@
+{
+    let i := 2
+    let x := 0x1234
+    mstore8(i, x)
+    let t := mload(0)
+    assert(eq(t, 0x0000340000000000000000000000000000000000000000000000000000000000))
```

Comment:
The test is failing. This seems to be
`0x340000000000000000000000000000000000000000000000000000000001`

Reply:
Oh right! Because memory is not zeroed initially!